spec/models/manageiq/providers/redfish/physical_infra_manager/operations/power_spec.rb

```diff
@@ -1,12 +1,15 @@
 describe ManageIQ::Providers::Redfish::PhysicalInfraManager do
-  let(:server) { FactoryBot.create(:redfish_physical_server, :vcr) }
   subject(:ems) do
     FactoryBot.create(:ems_redfish_physical_infra, :vcr)
   end
+  let(:server) do
+    FactoryBot.create(:redfish_physical_server, :vcr,
+                      :ext_management_system => ems)
+  end
 
   describe "#power_on", :vcr do
     it "powers on the system" do
-      ems.power_on(server, nil)
+      expect { ems.power_on(server, nil) }.to raise_error(MiqException::Error)
```

Comment:
Since we're doing refactoring, we could also make this second never used argument (`_options`) optional (`_options = nil`). Can be sometime later in a followup though.

Reply:
I will leave this for later. We still have some more work to do in this area, and maybe that parameter will become non-nil during the process of informing UI about the operation status.